### ISSUE TITLE
No error messages in python and no output

### ISSUE THREAD

**Issue by joanlyq** [HakaiInstitute/GlintMaskGenerator#33]
Hi there, I was trying to test the mask in python 3.8, installed the library with pip install with no issue in a conda env on MacOS. 

However, the test script doesn't give the output masks and there is no error messages printed either. 

Any help would appreciated. 

My test script is 
```
from glint_mask_generator import RGBThresholdMasker

masker = RGBThresholdMasker(img_dir="/path/to/my/images/", mask_dir="/path/to/store/masks/",thresholds=(0.9,0.9,0.9), pixel_buffer=3)
masker.process(max_workers=3, callback=print, err_callback=print)

```

I'm also attaching some test images below 
![DJI_0151](https://user-images.githubusercontent.com/72123513/212225187-6a9aaf86-f381-47f4-9142-575f71f6fd35.jpeg)
![DJI_0148](https://user-images.githubusercontent.com/72123513/212225155-b828a8d4-7640-495d-b474-5989c375cded.jpeg)
![DJI_0149](https://user-images.githubusercontent.com/72123513/212225183-a7cf328d-0fb7-488d-b68e-b3fdf67a307c.jpeg)
![DJI_0150](https://user-images.githubusercontent.com/72123513/212225185-c6f07f76-0542-4fc5-aebf-97e4adf789a9.jpeg)

**Comment by tayden** [comment]
Thanks for finding this! There was a bug in the code. 

It has been fixed in version 3.0.2. So you should just be able to update it with `pip install --upgrade glint-mask-tools` and your code will run.

**Comment by joanlyq** [comment]
Thx for the quick fix and it's working now!
